lib/linklist.c

```diff
@@ -92,6 +92,52 @@ void listnode_add_head(struct list *list, void *val)
 	list->count++;
 }
 
+bool listnode_add_sort_nodup(struct list *list, void *val)
+{
+	struct listnode *n;
+	struct listnode *new;
+	int             ret;
```

Comment:
Please follow the style guidelines.

Reply:
Shall change it and upload. checkpatch didn't catch it. I believed this way of alignment style shall give more readability to figure out variable names at ease.

Reply:
Yeah, I don't necessarily disagree, but we value consistency in style over anything.